test/DartTest.ts

```diff
@@ -41,8 +41,19 @@ describe('transpile to dart', () => {
       expectTranslate('var a;').to.equal(' var a ;');
       expectTranslate('var a:any;').to.equal(' dynamic a ;');
     });
-    it('should transpile variable declaration lists',
-       () => { expectTranslate('var a: number, b: string;').to.equal(' num a ; String b ;'); });
+    it('should transpile variable declaration lists', () => {
+      expectTranslate('var a, b;').to.equal(' var a , b ;');
+      expectTranslate('var a: number, b;').to.equal(' num a , b ;');
```

Comment:
Yes, as in other languages (C, Java, C#). I thought that that it is what it meant in TS, but apparently not. Very good catch!